container/bundle.bzl

```diff
@@ -37,9 +37,14 @@ def _container_bundle_impl(ctx):
 
     images = {}
     runfiles = []
+    if ctx.attr.stamp:
+        print("Attr 'stamp' is deprecated; it is now automatically inferred. Please remove it from %s" % ctx.label)
+    stamp = False
     for unresolved_tag in ctx.attr.images:
         # Allow users to put make variables into the tag name.
         tag = ctx.expand_make_variables("images", unresolved_tag, {})
+        if "{" in tag:
```

Comment:
can you please add docstring indicating in detail why we use a check for "{" to infer stamping (dont want that knowledge to be lost)